Makefile

```diff
@@ -164,24 +136,44 @@ endif
 # Build tsh against libfido2?
 # FIDO2=yes and FIDO2=static enable static libfido2 builds.
 # FIDO2=dynamic enables dynamic libfido2 builds.
-LIBFIDO2_MESSAGE := without libfido2
+LIBFIDO2_MESSAGE := without-libfido2
 ifneq (, $(filter $(FIDO2), yes static))
-LIBFIDO2_MESSAGE := with libfido2
+LIBFIDO2_MESSAGE := with-libfido2
 LIBFIDO2_BUILD_TAG := libfido2 libfido2static
 else ifeq ("$(FIDO2)", "dynamic")
-LIBFIDO2_MESSAGE := with libfido2
+LIBFIDO2_MESSAGE := with-libfido2
 LIBFIDO2_BUILD_TAG := libfido2
 endif
 
 # Enable Touch ID builds?
 # Only build if TOUCHID=yes to avoid issues when cross-compiling to 'darwin'
 # from other systems.
-TOUCHID_MESSAGE := without Touch ID
+TOUCHID_MESSAGE := without-Touch-ID
 ifeq ("$(TOUCHID)", "yes")
-TOUCHID_MESSAGE := with Touch ID
+TOUCHID_MESSAGE := with-Touch-ID
 TOUCHID_TAG := touchid
 endif
 
+# Enable libpcsclite for testing?
```

Comment:
Why does the "testing" build links dynamically and "non-testing" statically? If we want to avoid this dependency, we could just disable the tag from the build to not force everyone to install it.

Reply:
The `LIBPCSCLITE_TEST_TAG` build tag is only used by `make test-go` and `make -C build.assets test` to dynamically link to `libpcsclite` for tests. `LIBPCSCLITE_TEST_TAG` does not introduce the dependency when building binaries. This is the same logic used for `libfido2` with the `LIBFIDO2_TEST_TAG` tag.

This isn't necessary but in my opinion it's a nice piece of helper logic to test `libpcsclite` when dynamic linking is possible and static linking is not set up, primarily for local developement. Let me know if I'm missing something though.